components/automate-ui/src/app/pages/project/details/project-details.component.scss

```diff
@@ -67,42 +67,9 @@ form {
   }
 }
 
-thead {
```

Comment:
Where did all this CSS go?

Reply:
The base ([display: table])[https://github.com/chef/automate/blob/master/components/automate-ui/src/app/components/table/table.component.scss#L4] that the new component uses displays things in a very sane way. I removed a ton of css and things look more or less identical. I couldn't notice a difference. We can do a side by side comparison with acceptance or master but I was trying to do that as I went and literally couldn't notice the difference. This is what we did for the tokens table fwiw.

Reply:
In this specific instance, I think some of this CSS was redundant with parent css. Must have been written before it moved to the parent? I double-checked here since I found the CSS strange but I don't think how the output looks changed at all after removing this CSS. I'll triple-check in the morning though.

Reply:
Glad to hear your experience @tylercloke was what I was expecting you to find! (Does not always work out that well 🤞 )